-auto-approve (#1978)
hashicorp/waypoint: website/content/docs/getting-started.mdx

## Destroy

When you are done, run `destroy` to remove the application from Docker.
Notice that the `destroy` command requires confirmation to run.

```shell-session

> $ waypoint destroy  -auto-approve

Reviewer: ```suggestion
$ waypoint destroy -auto-approve
```

> When you are done, run `destroy` to remove the application from Docker.
> 
> ```shell-session

Reviewer: ```suggestion
When you are done, run `destroy` to remove the application from Docker.
Notice that the `destroy` command requires confirmation to run.

```shell-session
```

> $ waypoint destroy  -auto-approve```

Reviewer: ```suggestion
$ waypoint destroy  -auto-approve
```
```
